src/docs/faq.md

```diff
@@ -8,11 +8,11 @@ Please read this page before asking for help. Your help request will be pointed
 
 ### What OS should I use?
 
-- This depends on your use case. Only you can answer this question. <!-- Not ubuntu plz -->
+- This depends on your use case. Only you can answer this question.
 
-### Should I use rw-legacy or full rom?
+### Should I use RW_LEGCY or full rom?
 
-- This depends on what you are looking to use your device for. If you are planning on installing Windows or macOS you need to use full rom. Rw-legacy only supports linux.
+- This depends on what you are looking to use your device for. If you are planning on installing Windows or macOS you need to use full rom. RW_LEGACY only supports linux.
```

Comment:
RW_LEGACY also should ideally only be used for dual booting, because you can end up locked out if you wipe ChromeOS and the dev_boot_altfw flag gets cleared

Reply:
Aside from that we also have submarine which we could recommend if not dual booting

Reply:
I'd like to address this in a seperate PR